<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowMembersDialog</class>
 <widget class="QDialog" name="ShowMembersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>B Nazanin</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>لیست اعضای کتابخانه</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Persian" country="Iran"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tbl_members">
     <column>
      <property name="text">
       <string>کد عضو</string>
      </property>
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>نام</string>
      </property>
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>نام خانوادگی</string>
      </property>
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>تلفن</string>
      </property>
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>تاریخ عضویت</string>
      </property>
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>آدرس</string>
      </property>
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlay_buttons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="text">
        <string>حذف اعضای انتخاب‌شده</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="text">
        <string>بازگشت</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
